### PR TITLE
Fix UI routing

### PR DESCRIPTION
**Pull request by kakkoyun**
Fix and refactor UI routes probably got broken while rebasing.

### CODE REVIEW

**Comment by kakkoyun** [comment]
This is needed before re-routing traffic on stage.

cc @krasi-georgiev

File: internal/server/server.go [hunk 1]
```diff
@@ -65,19 +65,32 @@ func New(logger log.Logger, reg *prometheus.Registry, opts ...Option) Server {
...
+		})
+
+		// NOTICE: Following redirects added to be compatible with existing Read UI.
+		// Paths are explicitly specified to prevent unnecessary request to read handler.
+		for _, p := range []string{
+			"graph",
+			"stores",
+			"status",
+		} {
+			p := p
```
Comment (squat) [inline internal/server/server.go:89]: why do we need this extra declaration here? It looks to me like all uses of `p` are synchronous with the loop

Comment (kakkoyun) [reply internal/server/server.go:89]: If you see line 91, we are using it in a closure because of that usage we need to re-declare it in the loop scope.

Comment (squat) [reply internal/server/server.go:89]: ack yes I see it now

**Pull request merged**